src/exponentially_weighted.cpp

```diff
@@ -6,7 +6,8 @@ template <typename T>
 py::array_t<T> ExponentiallyWeightedMean(const py::array_t<T> data, T alpha) {
```

Comment:
please just set this instead
```suggestion
py::array_t<T> ExponentiallyWeightedMean(const py::array_t<T, py::array::c_style | py::array::forcecast> data, T alpha) {
```

Reply:
Oh, that's a really clean approach! Thanks for the guidance, José.

I'm not super familiar with pybind11 functionality as I've mostly played around with Rcpp, so it's nice to know.

Reply:
Thanks for the fix!